### ISSUE TITLE
`yarn run start-console` fails on Apple silicon

### ISSUE THREAD

**Issue by evanstoner** [openshift/console-plugin-template#58]
Attempting to follow the getting started guide for the first time using Apple M2 and a ROSA cluster. I have followed the instructions for `qemu-user-static` but still receive an architecture-related error running `yarn run start-console`. What am I missing? We are a Red Hat partner exploring the console plugin system. Thanks!

(This looks different from https://github.com/openshift/console-plugin-template/issues/10.)

```
$ yarn run start-console
yarn run v1.22.21
$ ./start-console.sh
Starting local OpenShift console...
API Server: https://api.XXX.YYY.ZZZ.openshiftapps.com:6443
Console Image: quay.io/openshift/origin-console:latest
Console URL: http://localhost:9000
Trying to pull quay.io/openshift/origin-console:latest...
Error: choosing an image from manifest list docker://quay.io/openshift/origin-console:latest: no image found in manifest list for architecture arm64, variant "v8", OS linux
error Command failed with exit code 125.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

System info:

```
$ podman version
Client:       Podman Engine
Version:      4.6.0
API Version:  4.6.0
Go Version:   go1.20.6
Git Commit:   38e6fab9664c6e59b66e73523b307a56130316ae
Built:        Thu Jul 20 16:13:05 2023
OS/Arch:      darwin/arm64

Server:       Podman Engine
Version:      4.7.2
API Version:  4.7.2
Go Version:   go1.21.1
Built:        Tue Oct 31 10:30:33 2023
OS/Arch:      linux/arm64
```

```
$ podman machine info
Host:
  Arch: arm64
  CurrentMachine: podman-machine-default
  DefaultMachine: podman-machine-default
  EventsDir: /var/folders/x3/jdkpfq_s301fv3208971rx080000gn/T/podman-run--1/podman
  MachineConfigDir: /Users/estoner/.config/containers/podman/machine/qemu
  MachineImageDir: /Users/estoner/.local/share/containers/podman/machine/qemu
  MachineState: Running
  NumberOfMachines: 1
  OS: darwin
  VMType: qemu
Version:
  APIVersion: 4.6.0
  Built: 1689883985
  BuiltTime: Thu Jul 20 16:13:05 2023
  GitCommit: 38e6fab9664c6e59b66e73523b307a56130316ae
  GoVersion: go1.20.6
  Os: darwin
  OsArch: darwin/arm64
  Version: 4.6.0
```

```
$ podman machine ssh    
Connecting to vm podman-machine-default. To close connection, use `~.` or `exit`
Fedora CoreOS 39.20231204.2.1
Tracker: https://github.com/coreos/fedora-coreos-tracker
Discuss: https://discussion.fedoraproject.org/tag/coreos

Last login: Fri Dec 29 09:37:21 2023 from 192.168.127.1
core@localhost:~$ rpm-ostree status
State: idle
Deployments:
● fedora:fedora/aarch64/coreos/testing
                  Version: 39.20231204.2.1 (2023-12-05T16:40:37Z)
               BaseCommit: c35b380d7da8041330c89473791e935d2472004c4ee5a1e4f2f685ae6b558abc
             GPGSignature: Valid signature by E8F23996F23218640CB44CBE75CF5AC418B8E74C
      RemovedBasePackages: moby-engine 24.0.5-1.fc39
          LayeredPackages: qemu-user-static

  fedora:fedora/aarch64/coreos/testing
                  Version: 39.20231204.2.1 (2023-12-05T16:40:37Z)
               BaseCommit: c35b380d7da8041330c89473791e935d2472004c4ee5a1e4f2f685ae6b558abc
             GPGSignature: Valid signature by E8F23996F23218640CB44CBE75CF5AC418B8E74C
      RemovedBasePackages: moby-engine 24.0.5-1.fc39
```

**Comment by evanstoner** [comment]
Temporary workaround is adding `--arch x86_64` to `podman run` command in `start-console.sh`.

**Comment by spadgett** [comment]
Thanks for letting us know @evanstoner 

@jhadvig Can someone on the team try to recreate? We might need to add `--arch x86_64` to the start-console.sh script. We should test both podman and docker on Apple silicon.

**Comment by spadgett** [comment]
/remove lifecycle/stale
Fixed by #62

Looks like this was fixed as part of the dependency update PR.
